docs/release_process.md

```diff
@@ -21,23 +62,26 @@ Example branch names:
     * release/1.0.x
     * release/1.1.x
 
-Backport PRs should be opened against the relevant release branch.
-
 ## Tagging a release
 
-You must do these things to formally release a version:
+You must do the following steps to formally release a version.
+
+In the release branch:
 
-1. Create a new release branch if one does not already exist.
 1. Mark the release in the CHANGELOG by replacing the `## Unreleased` header
    with `## [VERSION] date`.
 1. Update `bpftrace_VERSION_MAJOR`, `bpftrace_VERSION_MINOR`, and
    `bpftrace_VERSION_PATCH` in `CMakeLists.txt` to the target version.
-1. Tag a release. We do this in the github UI by clicking "releases" (on same line
-   as "commits"), then "Draft a new release". The tag version and release title
-   should be the same and in `vX.Y.Z` format. The tag description should
+1. Tag a release. We do this in the github UI by clicking "releases" (on same
+   line as "commits"), then "Draft a new release". The tag version and release
+   title should be the same and in `vX.Y.Z` format. The tag description should
    be the same as what you added to `CHANGELOG.md`.
-1. Check that automation picks up the new release and uploads release assets
-   to the release.
-1. If automation fails, please fix the automation for next time and also manually
-   build+upload artifacts by running `scripts/create-assets.sh` from bpftrace root
-   dir and attach the generated archives to the release.
+1. Check that automation picks up the new release and uploads release assets to
+   the release.
+1. If automation fails, please fix the automation for next time and also
+   manually build+upload artifacts by running `scripts/create-assets.sh` from
+   bpftrace root dir and attach the generated archives to the release.
+
+Once the release is out:
+1. Backport the `CHANGELOG`, `CMakeLists.txt`, and `flake.lock` changes from the
+   release branch to master.
```

Comment:
Why does only the release branch get the new LLVM? Wouldn't we want master to also have it during the release stabilization window?

Reply:
Good point. My goal was to avoid updating LLVM in master before all errors are fixed which would block the development but it makes sense to backport it once the new CI job passes on the release branch. Let me add that.